src/pages/AddItem.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { withFirestore } from 'react-firestore';
+
+const AddItem = ({ firestore }) => {
+  const [name, setName] = useState('');
+
+  // Send the new item to Firebase
+  const addItem = name => {
+    firestore.collection('items').add({ name });
+  };
+
+  // The state every time an event happens
+  const handleChange = event => {
+    setName(event.target.value);
+  };
+
+  // Handle the click of the Add Item button on the form
+  const handleSubmit = event => {
+    event.preventDefault();
+    addItem(name);
+    setName('');
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Add Item:
+        <input value={name} type="text" onChange={handleChange} />
+      </label>
+      <input type="submit" value="Add Item" />
```

Comment:
Any reason to use `input` vs `button` for submission?

Reply:
I was originally using a button by then when I was putzing around in the React docs, I saw their example was using an input and wanted to keep it close to the available docs in case participants wanted to investigate. What would you prefer? 

React form docs: https://reactjs.org/docs/forms.html#controlled-components

Reply:
I would normally say `button`, but I get where you're coming from. Mirroring the docs will lead to less confusion 👍